src/components/KeyblockMicroblocksTable.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <table>
+    <tr>
+      <th>
+        Hash
+        <hint-tooltip>
+          {{ keyblocksHints.microblockHash }}
+        </hint-tooltip>
+      </th>
+      <th>
+        Time
+        <hint-tooltip>
+          {{ keyblocksHints.time }}
+        </hint-tooltip>
+      </th>
+      <th>
+        Transactions Count
+        <hint-tooltip>
+          {{ keyblocksHints.transactionsCount }}
+        </hint-tooltip>
+      </th>
+    </tr>
+    <tr
+      v-for="microblock in microblocks.data"
+      :key="microblock.hash">
+      <td>
+        <app-link :to="`/micro-blocks/${microblock.hash}`">
```

Comment:
```suggestion
        <app-link :to="`/microblocks/${microblock.hash}`">
```

The link is incorrect

Reply:
fixed